tests/lib/collectors/gsheets.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests the Workspace logging timesketch processor."""
+
+import unittest
+
+from dftimewolf import config
+from dftimewolf.lib import errors
+from dftimewolf.lib import state
+from dftimewolf.lib.collectors import gsheets
+import mock
+
+INVALID_SHEET = {
+    'range':
+        'Sheet2!A1:Y1000',
+    'majorDimension':
+        'ROWS',
+    'values': [[
+        'message', 'timestamp', 'timestamp_desc ', 'extra_field_1 ',
+        'extra_field_2'
+    ], ['A message', '1331698658276340', 'Write time', 'foo ', 'bar'],
+               ['', '1331698658276340', 'create', 'dodo'],
+               ['sdsadasd', '', 'ddd', 'dodo', 'd']]
+}
+VALID_SHEET = {
+    'range':
+        'Sheet1!A1:Z1000',
+    'majorDimension':
+        'ROWS',
+    'values': [[
+        'message', 'timestamp', 'datetime', 'timestamp_desc ', 'extra_field_1 ',
+        'extra_field_2'
+    ],
+               [
+                   'A message', '1331698658276340', '2015-07-24T19:01:01+00:00',
+                   'Write time', 'foo ', 'bar'
+               ],
+               [
+                   '', '1331698658276340', '2016-07-24T19:01:01+00:00',
+                   'create', 'dodo'
+               ], ['sdsadasd', '', '', 'ddd', 'dodo', 'd']]
+}
```

Comment:
It would be great if these structures were formatted in the same way (indentation-wise, etc.). Feel free to add a `pylint: disable=line-too-long`, IMO it makes sense to have a spreadsheet row represented in a single line.

Reply:
Done